.gitmodules

```diff
@@ -1,15 +1,21 @@
-[submodule "packages/foundry/lib/ERC3156"]
-	path = packages/foundry/lib/ERC3156
+[submodule "lib/ERC3156"]
+	path = lib/ERC3156
 	url = https://github.com/alcueca/ERC3156
-[submodule "packages/foundry/lib/dss-interfaces"]
-	path = packages/foundry/lib/dss-interfaces
+[submodule "lib/dss-interfaces"]
+	path = lib/dss-interfaces
 	url = https://github.com/makerdao/dss-interfaces
-[submodule "packages/foundry/lib/forge-std"]
-	path = packages/foundry/lib/forge-std
+[submodule "lib/forge-std"]
+	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
-[submodule "packages/foundry/lib/yieldspace-tv"]
-	path = packages/foundry/lib/yieldspace-tv
+[submodule "/Users/alcueca/Code/vault-v2/lib/forge-std"]
```

Comment:
pretty sure you don't want an absolute path here, prob a foundry bug if they added it this way